src/main/java/gov/cms/madie/measure/utilities/qicore411/ResourceUtils.java

```diff
@@ -0,0 +1,44 @@
+package gov.cms.madie.measure.utilities.qicore411;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Resource;
+
+import cms.gov.madie.measure.exceptions.InternalServerException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+
+public class ResourceUtils {
+  public static String getData(String resource) {
+    try (InputStream inputStream = ResourceUtils.class.getResourceAsStream(resource)) {
+      if (inputStream == null) {
+        throw new InternalServerException("Unable to fetch resource " + resource);
+      }
+      return new String(inputStream.readAllBytes());
+    } catch (IOException e) {
+      throw new UncheckedIOException(e);
+    } catch (NullPointerException nullPointerException) {
+      throw new InternalServerException("Resource name cannot be null");
+    }
+  }
+
+  /**
+   * @param bundleResource Bundle resource
+   * @return r4 resource
+   */
+  public static Resource getResource(Bundle bundleResource, String resourceType) {
```

Comment:
Having both a system `Resource` and a FHIR `Resource` type in the same Resource Utility confused me for a time.